src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -20452,6 +20452,29 @@ static void Main()
             CompileAndVerify(new[] { source, s_collectionExtensions }, verify: Verification.Skipped, expectedOutput: "[0, 1], ");
         }
 
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/72762")]
+        public void RefStruct_04()
+        {
+            var source = """
+                using System.Collections;
+                using System.Collections.Generic;
+
+                dynamic d = null;
+                S s = [d];
+
+                ref struct S : IEnumerable<int>
+                {
+                    public IEnumerator<int> GetEnumerator() => throw null;
+                    IEnumerator IEnumerable.GetEnumerator() => throw null;
+                    public void Add<T>(T t) => throw null;
+                }
+                """;
+
+            // https://github.com/dotnet/roslyn/issues/72762 - There should be diagnostics for S implementing an interface, and
+            // there should also be a diagnostic that `s` cannot be used a recevier for a dynamic call.
```

Comment:
>a [](http://example.com/codeflow?start=73&length=1)

Typo?

Reply:
Was only there for when the test is unskipped. It does appear to not assert anymore as you mentioned, so I just put in the diagnostics and removed this comment.